cwms-data-api/src/main/java/cwms/cda/data/dto/project2/project/Projects.java

```diff
@@ -0,0 +1,148 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2024 Hydrologic Engineering Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package cwms.cda.data.dto.project2.project;
+
+import cwms.cda.api.errors.FieldException;
+import cwms.cda.data.dto.CwmsDTOPaginated;
+import cwms.cda.data.dto.Location;
+import cwms.cda.formatters.Formats;
+import cwms.cda.formatters.annotations.FormattableWith;
+import cwms.cda.formatters.json.JsonV2;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+
+@FormattableWith(contentType = Formats.JSON, formatter = JsonV2.class)
+public class Projects extends CwmsDTOPaginated {
```

Comment:
This is for cataloging and might not have fully defined Project instances based on what the DAO returns in the catalog cursor?

Reply:
Its for CDA to return a paged list of Locks.  Like if there were so many that we wanted CDA to return some and have the user request next-page for the rest

Reply:
Oh, I read "Projects" and assumed it was a listing of reservoir projects instead of project locks.

Reply:
No - you are right. Its just Project I have no idea why I put Lock in my reply.   Its just a collection of Project plus some holders for the paging fields.